### PR TITLE
km paired-tag docs

### PR DESCRIPTION
**Pull request by kayleemathews**
### Description

This PR adds an Overview doc for the Paired-Tag pipeline and updates pipeline version numbers in SS2 docs.

----

### Checklist 
If you can answer "yes" to the following items, please add a checkmark next to the appropriate checklist item(s) **and** notify our WARP documentation team by tagging either @ekiernan or @kayleemathews in a comment on this PR.

- [ ] Did you add inputs, outputs, or tasks to a workflow?
- [ ] Did you modify, delete or move: file paths, file names, input names, output names, or task names?
- [ ] If you made a changelog update, did you update the pipeline version number?

### CODE REVIEW

File: website/docs/Pipelines/PairedTag_Pipeline/README.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+| ​​PairedTagDemultiplex as demultiplex ([WDL](https://github.com/broadinstitute/warp/blob/develop/tasks/skylab/PairedTagUtils.wdl)) | UPStools | Task used to trim the 3-bp sample barcode from the read2 ATAC fastq files and stores it in the readname. | 
+| ATAC ([WDL](https://github.com/broadinstitute/warp/blob/develop/pipelines/skylab/multiome/atac.wdl) and [documentation](../ATAC/README)) | fastqprocess, bwa-mem, SnapATAC2 | Workflow used to analyze 10x single-cell ATAC data. |
+
+
+## Outputs
+
+| Output variable name | Filename, if applicable | Output format and description |
+|--- | --- | --- | 
+| pairedtag_pipeline_version_out | N.A. | String describing the version of the Paired-Tag pipeline used. |
+| bam_aligned_output_atac | `<input_id>_atac.bam` | BAM file containing aligned reads from ATAC workflow; contains sample and cell barcodes stored in the BB tag if `preindex` is “true”. |
```
Comment (kayleemathews) [inline website/docs/Pipelines/PairedTag_Pipeline/README.md:107]: @ekiernan Do you know if any of these filenames are different for Paired-Tag vs Multiome? I didn't update any pf these.

File: website/docs/Pipelines/PairedTag_Pipeline/README.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+
+## Consortia support
+This pipeline is supported by the [BRAIN Initiative](https://braininitiative.nih.gov/) (BICCN and BICAN). 
+
+If your organization also uses this pipeline, we would like to list you! Please reach out to us by contacting the [WARP Pipeline Development team](mailto:warp-pipelines-help@broadinstitute.org).
+
+
+## Acknowledgements
+We are immensely grateful to the members of the BRAIN Initiative (BICAN Sequencing Working Group) and SCORCH for their invaluable and exceptional contributions to this pipeline. Our heartfelt appreciation goes to [ADD CONTRIBUTORS] for their unwavering dedication and remarkable efforts.
```
Comment (kayleemathews) [inline website/docs/Pipelines/PairedTag_Pipeline/README.md:133]: @ekiernan Would you mind adding the specific people we should thank for their contributions to this pipeline?

File: website/docs/Pipelines/PairedTag_Pipeline/README.md [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+
+# Paired-Tag Overview
+
+| Pipeline Version | Date Updated | Documentation Author | Questions or Feedback |
+| :----: | :---: | :----: | :--------------: |
+| [PairedTag_v0.0.3](https://github.com/broadinstitute/warp/releases) | January, 2024 | Kaylee Mathews | Please file GitHub issues in warp or contact [documentation authors](mailto:warp-pipelines-help@broadinstitute.org) |
+
+## Introduction to the Paired-Tag workflow
+
+The [Paired-Tag workflow](https://github.com/broadinstitute/warp/blob/develop/pipelines/skylab/paired_tag/PairedTag.wdl) is an open-source, cloud-optimized pipeline developed in collaboration with the [BRAIN Initiative Cell Census Network](https://biccn.org/) (BICCN) and the BRAIN Initiative Cell Atlas Network (BICAN). It supports the processing of 10x 3' single-cell and single-nucleus gene expression (GEX) and histone modification data generated with the [10x Chromium Multiome assay](https://www.10xgenomics.com/products/single-cell-multiome-atac-plus-gene-expression).
```
Comment (ekiernan) [inline website/docs/Pipelines/PairedTag_Pipeline/README.md:14]:
```suggestion
The [Paired-Tag workflow](https://github.com/broadinstitute/warp/blob/develop/pipelines/skylab/paired_tag/PairedTag.wdl) is an open-source, cloud-optimized pipeline developed in collaboration with the [BRAIN Initiative Cell Census Network](https://biccn.org/) (BICCN) and the BRAIN Initiative Cell Atlas Network (BICAN). It supports the processing of 3' single-nucleus histone modification data (generated with the [paired-tag protocol](https://www.nature.com/articles/s41594-023-01060-1)]) and 10x gene expression (GEX) data generated with the [10x Chromium Multiome assay](https://www.10xgenomics.com/products/single-cell-multiome-atac-plus-gene-expression).

File: website/docs/Pipelines/PairedTag_Pipeline/README.md [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+
+The workflow is a wrapper WDL script that calls two subworkflows: the Optimus workflow for single-cell GEX data and the ATAC workflow for single-cell histone modification data.
```
Comment (ekiernan) [inline website/docs/Pipelines/PairedTag_Pipeline/README.md:16]:
```suggestion
The workflow is a wrapper WDL script that calls two subworkflows: the Optimus workflow for single-nucleus GEX data and the ATAC workflow for single-nucleus histone modification data.

File: website/docs/Pipelines/PairedTag_Pipeline/README.md [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+
+The [Optimus workflow](../Optimus_Pipeline/README) (GEX) corrects cell barcodes (CBs) and Unique Molecular Identifiers (UMIs), aligns reads to the genome, calculates per-barcode and per-gene quality metrics, and produces a raw cell-by-gene count matrix.
+
+The [ATAC workflow](../ATAC/README) (histone modification) corrects CBs, aligns reads to the genome, calculates per-barcode quality metrics, and produces a fragment file.
+
+The [wrapper WDL](https://github.com/broadinstitute/warp/blob/develop/pipelines/skylab/paired_tag/PairedTag.wdl) is available in the [WARP repository](https://github.com/broadinstitute/warp).
+
+:::info NOTE
+The Paired-Tag pipeline is under active development and has not been officially released or rigorously tested.
```
Comment (ekiernan) [inline website/docs/Pipelines/PairedTag_Pipeline/README.md:25]:
```suggestion
The Paired-Tag WDL is under active development (beta); it is not officially released and is undergoing scientific validation.

File: website/docs/Pipelines/PairedTag_Pipeline/README.md [hunk 6]
```diff
@@ -0,0 +1,138 @@
...
+| tar_star_reference | TAR file containing a species-specific reference genome and GTF for Optimus (GEX) pipeline. | File | 
+| annotations_gtf | GTF file containing gene annotations used for GEX cell metric calculation and ATAC fragment metrics; must match the GTF used to build the STAR aligner. | File |
+| ref_genome_fasta | Genome FASTA file used for building the indices. | File |
+| mt_genes | Optional file for the Optimus (GEX) pipeline containing mitochondrial gene names used for metric calculation; default assumes 'mt' prefix in GTF (case insensitive). | File |
+| tenx_chemistry_version | Optional integer for the Optimus (GEX) pipeline specifying the 10x version chemistry the data was generated with; validated by examination of the first read 1 FASTQ file read structure; default is "3". | Integer |
+| emptydrops_lower | Optional threshold for UMIs for the Optimus (GEX) pipeline that empty drops tool should consider for determining cell; data below threshold is not removed; default is "100". | Integer |
```
Comment (ekiernan) [inline website/docs/Pipelines/PairedTag_Pipeline/README.md:74]:
```suggestion
| emptydrops_lower | **Not used for single-nucleus data.** Optional threshold for UMIs for the Optimus (GEX) pipeline that empty drops tool should consider for determining cell; data below threshold is not removed; default is "100". | Integer |

File: website/docs/Pipelines/PairedTag_Pipeline/README.md [hunk 7]
```diff
@@ -0,0 +1,138 @@
...
+| ignore_r1_read_length | Optional boolean for the Optimus (GEX) pipeline indicating if the pipeline should ignore barcode chemistry check; if "true", the workflow will not ensure the `10x_chemistry_version` input matches the chemistry in the read 1 FASTQ; default is "false". | Boolean |
+| star_strand_mode | Optional string for the Optimus (GEX) pipeline for performing STARsolo alignment on forward stranded, reverse stranded, or unstranded data; default is "Forward". | String |
+| count_exons | Optional boolean for the Optimus (GEX) pipeline indicating if the workflow should calculate exon counts **when in single-nucleus (sn_rna) mode**; if "true" in sc_rna mode, the workflow will return an error; default is "false". | Boolean |
+| gex_whitelist | Optional file containing the list of valid barcodes for 10x multiome GEX data; default is "gs://gcp-public-data--broad-references/RNA/resources/arc-v1/737K-arc-v1_gex.txt". | File |
+| atac_r1_fastq | Array of read 1 paired-end FASTQ files representing a single 10x multiome ATAC library. | Array[File] |
```
Comment (ekiernan) [inline website/docs/Pipelines/PairedTag_Pipeline/README.md:80]:
```suggestion
| atac_r1_fastq | Array of read 1 paired-end FASTQ files representing a single paired-tag DNA library. | Array[File] |

File: website/docs/Pipelines/PairedTag_Pipeline/README.md [hunk 8]
```diff
@@ -0,0 +1,138 @@
...
+| atac_r2_fastq | Array of barcodes FASTQ files representing a single 10x multiome ATAC library. | Array[File] |
```
Comment (ekiernan) [inline website/docs/Pipelines/PairedTag_Pipeline/README.md:81]:
```suggestion
| atac_r2_fastq | Array of barcodes FASTQ files representing a single paired-tag DNA library. | Array[File] |

File: website/docs/Pipelines/PairedTag_Pipeline/README.md [hunk 9]
```diff
@@ -0,0 +1,138 @@
...
+| atac_r3_fastq | Array of read 2 paired-end FASTQ files representing a single 10x multiome ATAC library. | Array[File] |
```
Comment (ekiernan) [inline website/docs/Pipelines/PairedTag_Pipeline/README.md:82]:
```suggestion
| atac_r3_fastq | Array of read 2 paired-end FASTQ files representing a single paired-tag DNA library. | Array[File] |

File: website/docs/Pipelines/PairedTag_Pipeline/README.md [hunk 10]
```diff
@@ -0,0 +1,138 @@
...
+
+## Paired-Tag tasks and tools
+
+The Paired-Tag workflow calls two WARP subworkflows and an additional task which are described briefly in the table below. For more details on each subworkflow and task, see the documentation and WDL scripts linked in the table.
+
+| Subworkflow/Task | Software | Description | 
+| ----------- | -------- | ----------- |
+| Optimus ([WDL](https://github.com/broadinstitute/warp/blob/develop/pipelines/skylab/optimus/Optimus.wdl) and [documentation](../Optimus_Pipeline/README)) | fastqprocess, STARsolo, Emptydrops | Workflow used to analyze 10x single-cell GEX data. |
+| ​​PairedTagDemultiplex as demultiplex ([WDL](https://github.com/broadinstitute/warp/blob/develop/tasks/skylab/PairedTagUtils.wdl)) | UPStools | Task used to trim the 3-bp sample barcode from the read2 ATAC fastq files and stores it in the readname. | 
+| ATAC ([WDL](https://github.com/broadinstitute/warp/blob/develop/pipelines/skylab/multiome/atac.wdl) and [documentation](../ATAC/README)) | fastqprocess, bwa-mem, SnapATAC2 | Workflow used to analyze 10x single-cell ATAC data. |
```
Comment (ekiernan) [inline website/docs/Pipelines/PairedTag_Pipeline/README.md:99]:
```suggestion
ATAC ([WDL](https://github.com/broadinstitute/warp/blob/develop/pipelines/skylab/multiome/atac.wdl) and [documentation](../ATAC/README)) | fastqprocess, bwa-mem, SnapATAC2 | Workflow used to analyze single-nucleus paired-tag DNA (histone modifications) data. |

File: website/docs/Pipelines/PairedTag_Pipeline/README.md [hunk 11]
```diff
@@ -0,0 +1,138 @@
...
+
+
+## Paired-Tag tasks and tools
+
+The Paired-Tag workflow calls two WARP subworkflows and an additional task which are described briefly in the table below. For more details on each subworkflow and task, see the documentation and WDL scripts linked in the table.
+
+| Subworkflow/Task | Software | Description | 
+| ----------- | -------- | ----------- |
+| Optimus ([WDL](https://github.com/broadinstitute/warp/blob/develop/pipelines/skylab/optimus/Optimus.wdl) and [documentation](../Optimus_Pipeline/README)) | fastqprocess, STARsolo, Emptydrops | Workflow used to analyze 10x single-cell GEX data. |
+| ​​PairedTagDemultiplex as demultiplex ([WDL](https://github.com/broadinstitute/warp/blob/develop/tasks/skylab/PairedTagUtils.wdl)) | UPStools | Task used to trim the 3-bp sample barcode from the read2 ATAC fastq files and stores it in the readname. |
```
Comment (ekiernan) [inline website/docs/Pipelines/PairedTag_Pipeline/README.md:98]:
```suggestion
| ​​PairedTagDemultiplex as demultiplex ([WDL](https://github.com/broadinstitute/warp/blob/develop/tasks/skylab/PairedTagUtils.wdl)) | UPStools | Task used to trim the 3-bp sample barcode from the read2 ATAC fastq files and stores it in the readname. When this task is used, the ATAC workflow will add a combined 3 bp sample barcode and cellular barcode to the BB tag of the BAM. |

File: website/docs/Pipelines/PairedTag_Pipeline/README.md (hunk 1 above)
Comment (ekiernan) [reply website/docs/Pipelines/PairedTag_Pipeline/README.md:107]: Should be the same I think

File: website/docs/Pipelines/PairedTag_Pipeline/README.md [hunk 12]
```diff
@@ -0,0 +1,138 @@
...
+| ATAC ([WDL](https://github.com/broadinstitute/warp/blob/develop/pipelines/skylab/multiome/atac.wdl) and [documentation](../ATAC/README)) | fastqprocess, bwa-mem, SnapATAC2 | Workflow used to analyze 10x single-cell ATAC data. |
+
+
+## Outputs
+
+| Output variable name | Filename, if applicable | Output format and description |
+|--- | --- | --- | 
+| pairedtag_pipeline_version_out | N.A. | String describing the version of the Paired-Tag pipeline used. |
+| bam_aligned_output_atac | `<input_id>_atac.bam` | BAM file containing aligned reads from ATAC workflow; contains sample and cell barcodes stored in the BB tag if `preindex` is “true”. |
+| fragment_file_atac | `<input_id>_atac.fragments.sorted.tsv.gz` | Sorted and bgzipped TSV file containing fragment start and stop coordinates per barcode. The columns are "Chromosome", "Start", "Stop", "ATAC Barcode", "Number of reads", and "GEX Barcode". |
```
Comment (ekiernan) [inline website/docs/Pipelines/PairedTag_Pipeline/README.md:108]: This one is not sorted. We don't include that task

Comment (ekiernan) [inline website/docs/Pipelines/PairedTag_Pipeline/README.md:108]:
```suggestion
| fragment_file_atac | `<input_id>_atac.fragments.tsv` or if preindexing = true, `<input_id>_atac.fragments.BB.tsv | TSV file containing fragment start and stop coordinates per barcode. The columns are "Chromosome", "Start", "Stop", "Barcode", and "Number of reads". |

File: website/docs/Pipelines/PairedTag_Pipeline/README.md [hunk 13]
```diff
@@ -0,0 +1,138 @@
...
+| cell_metrics_gex | `<input_id>_gex.cell_metrics.csv.gz` | CSV file containing the per-cell (barcode) metrics. |
+| gene_metrics_gex | `<input_id>_gex.gene_metrics.csv.gz` | CSV file containing the per-gene metrics. |
+| cell_calls_gex | `<input_id>_gex.emptyDrops` | TSV file containing the EmptyDrops results when the Optimus workflow is run in sc_rna mode. |
+| h5ad_output_file_gex | `<input_id>_gex.h5ad` | h5ad (Anndata) file containing the raw cell-by-gene count matrix, gene metrics, cell metrics, and global attributes. See the [Optimus Count Matrix Overview](../Optimus_Pipeline/Loom_schema.md) for more details. |
+
+
+## Versioning and testing
+
+All Paired-Tag pipeline releases are documented in the [Paired-Tag changelog](https://github.com/broadinstitute/warp/blob/develop/pipelines/skylab/paired_tag/PairedTag.wdl) and tested using [plumbing and scientific test data](https://github.com/broadinstitute/warp/tree/develop/pipelines/skylab/paired_tag/test_inputs). To learn more about WARP pipeline testing, see [Testing Pipelines](https://broadinstitute.github.io/warp/docs/About_WARP/TestingPipelines).
```
Comment (ekiernan) [inline website/docs/Pipelines/PairedTag_Pipeline/README.md:123]:
```suggestion
All Paired-Tag pipeline releases are documented in the [Paired-Tag changelog](https://github.com/broadinstitute/warp/blob/develop/pipelines/skylab/paired_tag/PairedTag.wdl) and tested using [plumbing and scientific test data](https://github.com/broadinstitute/warp/tree/develop/pipelines/skylab/paired_tag/test_inputs). To learn more about WARP pipeline testing, see [Testing Pipelines](https://broadinstitute.github.io/warp/docs/About_WARP/TestingPipelines). Note that paired-tag tests are still in development.

File: website/docs/Pipelines/PairedTag_Pipeline/README.md (hunk 2 above)
Comment (ekiernan) [inline website/docs/Pipelines/PairedTag_Pipeline/README.md:133]:
```suggestion
We are immensely grateful to the members of the BRAIN Initiative (BICAN Sequencing Working Group) and SCORCH for their invaluable and exceptional contributions to this pipeline. Our heartfelt appreciation goes to Dr. Bing Ren's lab, Yang Xie, and Lei Chang for their unwavering dedication and remarkable efforts.

**Review by ekiernan — APPROVED** [review]
Thank you! This is looking great. Made a few clarifications.

**Review by kevinpalis — APPROVED** [review]
LGTM

**Pull request merged**